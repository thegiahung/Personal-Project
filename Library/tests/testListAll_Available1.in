Add book books.csv 111111
list all
list all long
list available
list available long
exit
