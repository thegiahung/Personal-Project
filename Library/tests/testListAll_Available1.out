user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: To Kill a Mockingbird (Harper Lee)

user: 111111: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

user: To Kill a Mockingbird (Harper Lee)

user: 111111: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

user: Ending Library process.
